dali/base/dasds.cpp

```diff
@@ -3241,6 +3254,7 @@ PDState CServerRemoteTree::checkChange(IPropertyTree &changeTree, CBranchChange
                 if (!t)
                     t = changeTree.addPropTree(ATTRDELETE_TAG, createPTree());
                 t->addProp(EXT_ATTR, "");
+                t->setProp(COMPRESS_ATTR, nullptr);//MORE: What should this do?
```

Comment:
I think reasonable. This will have the affect of deleting COMPRESS_ATTR too (as well as EXT_ATTR).
Both should use nullptr vs "" really.
For reference this is the server-side handling code:
```
            IPropertyTree *ad = change.queryPropTree(ATTRDELETE_TAG);
            if (ad)
            {
                Owned<IAttributeIterator> aIter = ad->getAttributes();
                ForEach (*aIter)
                {
                    if (!currentBranch.removeProp(aIter->queryName()))
                        OWARNLOG("Property '%s' missing, but recorded as being present at time of delete, in section '%s'", aIter->queryName(), headerPath.get());
                }
            }
```